python/ql/src/Security/CWE-089/SqlInjection.qhelp

```diff
@@ -21,20 +21,29 @@ or prepared statements.
 
 <example>
 <p>
-In the following snippet, from an example django app, 
-a name is stored in the database using two different queries.
+In the following snippet, an user is fetched from the database using three
```

Comment:
Apologies for the vagaries of the English language. When the vowel "u" is pronounced as a "y", apparently it's always "a" rather than "an".
```suggestion
In the following snippet, a user is fetched from the database using three
```

Reply:
Thanks! I guess I mechanically applied the vowel requires `an` rule. I'll try to see if I can also get a grammar checker, in addition to my newly acquired spell checker :wink: